src/OpenTracing/Tracer.php

```diff
@@ -20,7 +20,7 @@ public function getScopeManager();
      * Tracer::getScopeManager()->getActive()->getSpan(),
      * and null will be returned if {@link Scope#active()} is null.
      *
-     * @return Span|null
+     * @return ScopedSpan|null
```

Comment:
ScopedSpan class does not exist. It should return a Span.

Reply:
Totally. I was playing around with the idea of having such a thing but I decided to not to in the end.